docs_user/modules/openstack-mariadb_copy.adoc

```diff
@@ -73,18 +129,17 @@ SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' Mysql
 ----
 for i in "${!SOURCE_GALERA_MEMBERS[@]}"; do
   echo "Checking for the database node $i WSREP status Synced"
-  sudo podman run -i --rm --userns=keep-id -u $UID $MARIADB_IMAGE mysql \
+  oc exec mariadb-copy-data -- bash -c 'mysql \
```

Comment:
please use " so vat env vars got interpolated

Reply:
or you can as well use rsh https://github.com/openstack-k8s-operators/data-plane-adoption/pull/316/files#diff-d5dba85fef50c60b9a5a0ead4321eea3518b41e38cb65892376198a8cd23c1c8R77 ?

Reply:
Doc changes will be reflected in next PR